.github/workflows/minimal-app.yml

```diff
@@ -95,6 +96,9 @@ jobs:
       - run: docker push ${{ steps.naming-selector.outputs.FULL_IMAGE_NAME }}
```

Comment:
It looks like you tag latest but don't push latest. This step could be multiple parts in the same step similar to line 59 where it sets the variables since it all essentially does 1 thing which is pushing images. 

```
- run: |
    docker tag ${{ steps.naming-selector.outputs.FULL_IMAGE_NAME }} ${{ steps.naming-selector.outputs.LATEST_FULL_IMAGE_NAME }}
    docker push ${{ steps.naming-selector.outputs.FULL_IMAGE_NAME }}
    docker push ${{ steps.naming-selector.outputs.LATEST_FULL_IMAGE_NAME }}
  if: env.needs_release == 'true'
```

Reply:
Changed